src/packages/cli/fixtures/test.sh

```diff
@@ -5,7 +5,7 @@
 # Test version command
 #
 VERSION=$(node ./dist/bin.js --version)
-if [[ ${VERSION} != *"@prisma/cli"* ]]; then
+if [[ ${VERSION} != *"@prisma/client"* ]]; then
```

Comment:
don't think this should have been renamedd

Reply:
It's because it's the output of `npx prisma -v`

We can rely on having `@prisma/client` always present there.